tasks/mesos.yml

```diff
@@ -104,4 +107,5 @@
 - name: Control status of mesos-slave service
   service:
     name: mesos-slave
-    state: "{% if 'slave' in mesos_install_mode %}{% if mesos_slave_config_updated or force_restart|bool %}re{% endif %}started{% else %}stopped{% endif %}"
+    state: "{% if mesos_slave_config_updated or force_restart|bool %}re{% endif %}started"
```

Comment:
already better to read with part of the logic in the `when`
In *general*, I still prefer the ternary pattern (but we can wait for the upstream maintainer's opinion), that would result in:
```
    state: "{{ mesos_slave_config_updated or force_restart|ternary('restarted','started')}}"
```